ramls/order.raml

```diff
@@ -31,19 +31,12 @@ traits:
   searchable: !include raml-util/traits/searchable.raml
   language: !include raml-util/traits/language.raml
   validate: !include raml-util/traits/validation.raml
-  auth:
-    responses:
-      401:
-        description: "Not authorized to perform requested action"
-        body:
-          text/plain:
-            example: "unable to create <<resourcePathName|!singularize>> -- unauthorized"
 
 resourceTypes:
-  collection: !include raml-util/rtypes/collection.raml
-  collection-item: !include raml-util/rtypes/item-collection.raml
-  collection-get: !include raml-util/rtypes/collection-get.raml
-  post-with-200: !include rtypes/post-json-body-200.raml
+  collection: !include rtypes/collection-with-json-response.raml
```

Comment:
The [RMB's RestVerticle](https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java#L832) returns `application/json` error response only for 422 status. In case `RestVerticle` returns the response without invocation of the module's business logic methods, the `text/plain` will be returned back to the caller.

Reply:
@piotr-kalashuk I didn't realize this.  I feel as though this makes the API a little confusing... Now clients will need to know how to handle an application/json response as well as as text/plain response.  I guess it is what it is for now.  I'll take this up with the RMB team